content/en/logs/processing/attributes_naming_convention.md

```diff
@@ -231,6 +231,24 @@ These attributes are related to the data added by a syslog or a log-shipper agen
 
 Some integrations that rely on these are: [Rsyslog][15], [NxLog][16], [Syslog-ng][17], [Fluentd][18], [Logstash][19], etc.
 
+### DNS
+
+All attributes and measures are prefixed by `dns`.
+
+| **Fullname**         | **Type** | **Description**                                                           |
+|:---------------------|:---------|:--------------------------------------------------------------------------|
+| `dns.id`             | `string` | The DNS query identifier.                                                 |
+| `dns.question.name`  | `string` | The IP address URL that the DNS question wishes to find.                         |
+| `dns.question.type`  | `string` | A two octet code which specifies the DNS question type (example: "AAAA").  |
```

Comment:
The 2 octet code is what is present in the DNS packet but here it is in human readable form (same for the answer type)
As a reference: https://en.wikipedia.org/wiki/List_of_DNS_record_types

Reply:
added ref link